kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -150,27 +124,19 @@
         type: Boolean,
         default: false,
       },
-      contentId: {
-        type: String,
-        default: null,
-      },
-      contentNode: {
+      content: {
```

Comment:
When objects are used as props it can be valuable to add validator to ensure the shape of the data is correct.

Some examples from the codebase:

https://github.com/learningequality/kolibri/blob/c3238a85915a75386150c5f1191de6735e495d1c/kolibri/core/assets/src/views/sync/FacilityAdminCredentialsForm.vue#L54-L60

https://github.com/learningequality/kolibri/blob/c3238a85915a75386150c5f1191de6735e495d1c/kolibri/plugins/setup_wizard/assets/src/views/importFacility/SelectFacilityForm.vue#L67-L73

https://github.com/learningequality/kolibri/blob/c3238a85915a75386150c5f1191de6735e495d1c/kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/SelectGroup.vue#L72-L79

I'll add a follow-up issue on looking at a way to standardize these and avoid the ad-hoc nature of them